src/Config/BaseProjectConfig.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace ShoppinPal\YapepCommon\Config;
+
+use YapepBase\Exception\Exception;
+
+abstract class BaseProjectConfig
+{
+    /**
+     * @param string $configName
+     * @param mixed  $default
+     *
+     * @return mixed
+     *
+     * @throws Exception If the value is not set and no default is provided.
+     */
+    public static function getEnvConfigValue($configName, $default = null)
+    {
+        // Remove all config values from the $_SERVER superglobal so we don't have secrets in the logs
```

Comment:
I'm not sure about this.
First of all this logic has nothing to do in a simple getter for multiple purposes:
- It's not about getting a value
- If you want to delete everything from the $_SERVER what comes from the ENV and you are not getting everything then you did only a half job
Second of all it can happen that you need these details for debugging purposes. For example connection details and such

Reply:
The problem is, we are dumping the server array into the debug logs. This means our database passwords are in plain text in the trace files, getting uploaded to S3.

I agree, this is not about getting a value, but we don't really have any other efficient way to remove secrets from the server array. I can't just nuke the server array, because it may be useful for other reasons.

The only other way would be to remove the data from the debug information when creating the trace file, but that's not very reliable, and also doesn't really belong there, as the debugger shouldn't really know about our configuration in the env files, so parsing the .env.example file is not the greatest idea. Another option would be to use whitelisting instead, and add every possible $_SERVER key that is possibly useful, and remove everything else from it...